### ISSUE TITLE
Integration with ROS

### ISSUE THREAD

**Issue by pushkalkatara** [AlexeyAB/darknet#1516]
There is darknet_ros version for darknet here - https://github.com/leggedrobotics/darknet_ros
But this is very slow compared to this version of the darknet.
I was trying to integrate this version with the same methodology but facing some compilation issues.
Here's the link - https://github.com/pushkalkatara/darknet_ros

**Comment by MilkWithCoffee** [comment]
Hi everyone,

please find my fork of the darknet_ros repo (https://github.com/leggedrobotics/darknet_ros) using **AlexeyAB's version of darknet** here:
https://github.com/MilkWithCoffee/darknet_ros

With that, you should be easily able to run this sweety with ROS.

**Comment by pushkalkatara** [comment]
@MilkWithCoffee there are more optimizations possible. Like the letterbox image technique is used, but in the AlexeyAB's repository 3 different resizing techniques are available-
https://github.com/AlexeyAB/darknet/issues/232#issuecomment-336955485

**Comment by MilkWithCoffee** [comment]
> @MilkWithCoffee there are more optimizations possible. Like the letterbox image technique is used, but in the AlexeyAB's repository 3 different resizing techniques are available-
> [#232 (comment)](https://github.com/AlexeyAB/darknet/issues/232#issuecomment-336955485)

Thanks for the hint. Even though I didn´t want to go too far from Alexey´s great work, I will check it out.

**Comment by pushkalkatara** [comment]
Hi, I adapted the alexby version to a ROS package. Link - https://github.com/pushkalkatara/darknet_ros

**Comment by fiorano10** [comment]
Did anyone make a ros wrapper compatible with this Darknet repo? w/o tracking

**Comment by pushkalkatara** [comment]
Hi @fiorano10 Yes there is a ROS wrapper for the darknet repo as well. You can find it on this link - https://github.com/leggedrobotics/darknet_ros

**Comment by fiorano10** [comment]
I asked ‘this’, as in alexey darknet ros repo

**Comment by pushkalkatara** [comment]
Yes, there's a darknet wrapper for alexeby repo too - https://github.com/pushkalkatara/darknet_ros

**Comment by fiorano10** [comment]
@pushkalkatara without tracking. [issue#6](https://github.com/pushkalkatara/darknet_ros/issues/6)

**Comment by Shame-fight** [comment]
> Did anyone make a ros wrapper compatible with this Darknet repo? w/o tracking

I also have the same needs

**Comment by Shame-fight** [comment]
> Did anyone make a ros wrapper compatible with this Darknet repo? w/o tracking



> Did anyone make a ros wrapper compatible with this Darknet repo? w/o tracking

I also have the same need to integrate this version of darknet with ros
@AlexeyAB @fiorano10 @pushkalkatara

**Comment by alexandrosnic** [comment]
have you guys find an integration of Alexey's darknet ROS (without tracking)?
@fiorano10 @Shame-fight
